common/app/conf/switches/CommercialSwitches.scala

```diff
@@ -86,6 +86,15 @@ trait CommercialSwitches {
     exposeClientSide = true
   )
 
+  val LiveblogDynamicAdvertsSwitch = Switch(
+    "Commercial",
+    "liveblog-dynamic-adverts",
+    "Dynamically insert inline adverts on liveblogs",
+    safeState = Off,
+    sellByDate = new LocalDate(2016, 2, 14),
```

Comment:
@kelvin-chappell argh, I looked at my calendar, but I'm still used to Mo-Su weeks, not Su-Sa like it is here in the UK :'(